django_rq/workers.py

```diff
@@ -23,3 +23,20 @@ def get_worker(*queue_names):
     return Worker(queues,
                   connection=queues[0].connection,
                   exception_handlers=get_exception_handlers() or None)
+
+
+def collect_workers_per_configuration(queue_list):
```

Comment:
One more thing, do you mind documenting this part more extensively, including an example of how workers_collections look like?